server/dbs/__init__.py

```diff
@@ -17,9 +17,10 @@
 #  limitations under the License.
 ##############################################################################
 
-from base import (getDBConnectorClass, getDBConnector,   # noqa
-                  clearDBConnectorCache, FilterOperators,
-                  DatabaseConnectorException)
-import sqlalchemydb  # noqa
-import postgres_sqlalchemy  # noqa
-import mongo  # noqa
+from .base import (   # noqa
+    getDBConnectorClass, getDBConnector, getDBConnectorClassFromDialect,
+    clearDBConnectorCache, FilterOperators, DatabaseConnectorException,
+    databaseFromUri)
+from . import sqlalchemydb  # noqa
+from . import postgres_sqlalchemy  # noqa
+from . import mongo  # noqa
```

Comment:
Adding an `__all__ =` line would prevent the local unused errors.  I think that will take care of it.